mypy/suggestions.py

```diff
@@ -135,19 +142,28 @@ def is_implicit_any(typ: Type) -> bool:
 class SuggestionEngine:
     """Engine for finding call sites and suggesting signatures."""
 
-    def __init__(self, fgmanager: FineGrainedBuildManager):
+    def __init__(self, fgmanager: FineGrainedBuildManager,
+                 json: bool,
+                 no_errors: bool = False,
+                 no_any: bool = False) -> None:
```

Comment:
It seems inconsistent to have defaults for the new flags but not for json. It seems that at runtime all flags are always passed by the caller; OTOH it may be more consistent with the spirit of `**kwds` to have a default for `json` too.